test/secret_auth.sh

```diff
@@ -33,6 +33,35 @@ testcase_token_works_for_v1_api() {
     assert_equal '[]' $stdout
 }
 
+testcase_input_can_be_cancelled() {
+    setup
+
+    # Run bacalhau job list in a screen session, else the auth code will detect
+    # that a TTY is not in use and won't use the hanging code
+    CMD='bacalhau job list --output=json'
+    rm -f screenlog.0
+    screen -L -d -m bacalhau job list --output=json
```

Comment:
Is it safer to use `$BACALHAU` here so that we can use it locally (at least for me this might invoke the system installed version in /usr/local/bin

Reply:
The built `bacalhau` is selected by virtue of appearing first on the `$PATH`... do you mean that `screen` might ignore the contents of `$PATH`?

Reply:
Ah no, if it's first on path I'm less worried.